lib/client.js

```diff
@@ -33,6 +33,11 @@ const debugQuery = (sql, txId) => _debugQuery(sql.replace(/%/g, '%%'), txId);
 
 const { POOL_CONFIG_OPTIONS } = require('./constants');
 
+const ReuseConnection = (connection) =>
+  async function withConnection(next) {
+    return next(connection);
+  };
+
```

Comment:
Oops -- I thought I deleted this part already.  I'll fix this in a little while.  (I currently have the tests running on loop so that we can detect unhandled promise rejections)